src/client/java/teammates/client/scripts/DataMigrationForRubricQuestionsWithWeightsAttached.java

```diff
@@ -93,20 +95,25 @@ protected void postAction() {
         // nothing to do
     }
 
-    protected FeedbackQuestion updateWeights(FeedbackQuestion question) {
+    protected void updateWeights(FeedbackQuestion question) {
         FeedbackQuestionAttributes attr = FeedbackQuestionAttributes.valueOf(question);
         FeedbackRubricQuestionDetails fqd = (FeedbackRubricQuestionDetails) attr.getQuestionDetails();
         List<List<Double>> weightsForEachCell = fqd.getRubricWeights();
+        FeedbackQuestionsDb fqDb = new FeedbackQuestionsDb();
+
         try {
             Field rubricWeightsForEachCell = fqd.getClass().getDeclaredField("rubricWeightsForEachCell");
             rubricWeightsForEachCell.setAccessible(true);
             rubricWeightsForEachCell.set(fqd, weightsForEachCell);
 
             attr.setQuestionDetails(fqd);
-            return attr.toEntity();
+            fqDb.updateFeedbackQuestion(attr);
```

Comment:
The things is that fqDB will fetch the question again and update the attribute.

If we can just set the serialised JSON by `question.setQuestionText(attr.questionMetaData);`, why we need `fqDB`?

Reply:
I haven't tried setting the JSON using `question.setQuestionText(attr.questionMetaData);` yet. Will add some rubric question by checking out an older commit to try this approach.

Reply:
That is part of the operations which `fqDb.updateFeedbackQuestion(attr);` does.

Maybe you could also add `fq.keepUpdateTimestamp = keepUpdateTimestamp;` to prevent timestamp changes.